src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -3431,68 +3431,133 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, post_ops_optimizations_input_range, ::test
     convolution_test_params{ CASE_CONV_S8S8_15, 2, 3 },
 }));
 
+struct convolution_eltw_sum_test_params {
+    tensor in_shape;
+    tensor out_shape;
+    tensor kernel;
+    tensor stride;
+    tensor pad;
+    tensor dilation;
+    uint32_t groups;
+    data_types data_type;
+    format input_format;
+    data_types weights_type;
+    format weights_format;
+    data_types eltw_type;
+    format eltw_format;
+    data_types out_type;
+    format out_format;
+    data_types default_type;
+    format default_format;
+    size_t expected_fused_primitives;
+    size_t expected_not_fused_primitives;
+};
+
+class EltwiseSumFusingTestOneDNN : public BaseFusingTest<convolution_eltw_sum_test_params> {
+public:
+    void execute(convolution_eltw_sum_test_params& p) {
+        // Onednn post operation has issue in a machine that does not support imad.
```

Comment:
Why does it matter? If onednn is not used, the topology should run through cldnn.

Reply:
I will remove this condition.